soh/src/overlays/actors/ovl_En_Ishi/z_en_ishi.c

```diff
@@ -330,6 +330,12 @@ void EnIshi_Init(Actor* thisx, GlobalContext* globalCtx) {
         Actor_Kill(&this->actor);
         return;
     }
+    // If dungeon entrance randomizer is on, remove the grey boulders that normally
+    // block child Link from reaching the Fire Temple entrance.
+    if (type == ROCK_LARGE && gSaveContext.n64ddFlag && Randomizer_GetSettingValue(RSK_SHUFFLE_DUNGEON_ENTRANCES) &&
+        globalCtx->sceneNum == 0x061) { // Death Mountain Creater
```

Comment:
Does this not remove all rocks in DMC? EG, the ones blocking fairy fountain and the DMC scrub grotto

Reply:
ROCK_LARGE is silver gauntlet rocks, so it doesn't affect the grotto or fairy fountain that are hammer rocks (this is how 3ds rando does it).